### PR TITLE
Add content around communication and setup instructions

### PR DESCRIPTION
**Pull request by lisafrench**

### CODE REVIEW

**Comment by sburns** [comment]
❤️

File: docs/how_we_communicate.md [hunk 1]
```diff
@@ -1,44 +1,94 @@
...
+- We use [ZenHub](http://zenhub.io) for managing ticket priority.
+- We measure GitHub ticket estimations based on effort/difficulty/complexity,
+  more so than based on time to complete the ticket. We use this scale for
+  assigning GitHub ticket estimations:
+    - 0.5 = a few minutes
+    - 1 = half a day
+    - 2 = two days
+    - 3 = half a week
+    - 5 = a week
+    - 8 = around two weeks
```
Comment (briandailey) [inline docs/how_we_communicate.md:32]: :+1:

**Comment by briandailey** [comment]
@lisafrench this is great. Thanks!

**Comment by lisafrench** [comment]
@briandailey Thanks!

**Pull request merged**